docs/users-guide/performance/chunking.md

```diff
@@ -0,0 +1,65 @@
+# Chunking
+
+The default operation of ActivitySim is to attempt to run simulations in each
+component for that component's entire pool of choosers in a single operation.
+This allows for efficient use of vectorization to speed up computations, but can
+also lead to memory issues if the pool of choosers is too large.  This is particularly
+a problem in interaction-type models, where a large pool of choosers is faced
+with a large set of alternatives.
+
+ActivitySim includes the ability to "chunk" these model components into more
+manageable sized groups of choosers, which can be processed one chunk at a time.
+There is a small overhead associated with chunking, but if the total number of
+chunks is relatively small, the overhead is usually outweighed by the benefits
+in reduced memory usage.
+
+Chunking can be used in two ways in ActivitySim: dynamic and explicit.
```

Comment:
Suggest we recommend explicit chunking here.

Reply:
Yes, done.